proposal/policyexception-interfaces.md

```diff
@@ -0,0 +1,38 @@
+# Interfaces Design for operating PolicyExceptions in Kyverno
+
+```go
+type PolicyExceptionInterface interface {
+	metav1.Object
```

Comment:
What is this `metav1.Object` for?

Reply:
For calling k8s API?
would import 
```
metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Reply:
Say I'm the informer, can I use this interface to implement `AddFunc`? (And how if necessary...)